docs/source/schemas/best_practices.rst

```diff
@@ -37,3 +37,11 @@ Range Scans
    - Range scans inherently lead to hotspotting, because keys that are close to one another in a range will end up on the same Cassandra node.
 - When using dynamic columns for range scans, do not allow the total size of your row to grow above ~100MB.
    - Large rows reduce performance of compactions and create heap pressure in Cassandra
+
+Stream Stores
```

Comment:
Please add this to http://palantir.github.io/atlasdb/html/schemas/streams.html.

Reply:
Right, sounds good. I didn't initially do this as it is mentioned in the table, but will add it.